_includes/v19.1/known-limitations/cdc.md

```diff
@@ -6,4 +6,4 @@ The following are limitations in the v19.1 release and will be addressed in the
 - Partial or intermittent sink unavailability may impact changefeed stability; however, [ordering guarantees](change-data-capture.html#ordering-guarantees) will still hold for as long as a changefeed [remains active](change-data-capture.html#monitor-a-changefeed).
 - Changefeeds cannot be altered. To alter, cancel the changefeed and [create a new one with updated settings from where it left off](create-changefeed.html#start-a-new-changefeed-where-another-ended).
 - Additional target options will be added, including partitions and ranges of primary key rows.
-- In v19.1.0, when emitting deletes, [cloud storage sinks](create-changefeed.html#cloud-storage-sink) do not emit the record's keys; therefore, the deleted record is not identifiable.
+- In v19.1.0, when emitting deletes, [cloud storage sinks](create-changefeed.html#cloud-storage-sink) do not emit the record's keys; therefore, the deleted record is not identifiable. This has been fixed in v19.1.1.
```

Comment:
In situations like this I like to say a little more explicitly "in versions XXX and above" (if true) so the content continues to make sense even if it hangs around for a little while after 19.1.1 doesn't matter that much anymore.